src/AuthEncoding/AuthEncoding.py

```diff
@@ -162,6 +162,42 @@ def validate(self, reference, attempt):
 registerScheme(u'SHA256', SHA256DigestScheme())
 
 
+# Bcrypt support may not have been requested at installation time
+# - installed via the 'bcrypt' extra
+try:
+    import bcrypt
+except ImportError:
+    bcrypt = None
+
+
+if bcrypt is not None:
```

Comment:
I believe the parameterized tests do cover everything for the new handler (at least as much as they do for all the other handlers)